### PR TITLE
Added Confirm buy/sell/try luck menu.

### PR DESCRIPTION
**Pull request by oylsister**

### CODE REVIEW

File: addons/sourcemod/scripting/shop.sp [hunk 1]
```diff
@@ -1198,6 +1207,76 @@ public int ItemPanel_Handler(Menu menu, MenuAction action, int param1, int param
 	}
 }
 
+public Action ConfirmBuy(int client, int item_id)
+{
+	Menu menu = new Menu(Menu_ConfirmBuy);
+	
+	menu.SetTitle("Are you sure that you going to purchase this item?");
```
Comment (R1KO) [inline addons/sourcemod/scripting/shop.sp:1214]: translations?

File: addons/sourcemod/scripting/shop/functions.sp [hunk 2]
```diff
@@ -284,6 +284,47 @@ public int Functions_Menu_Handler(Menu menu, MenuAction action, int param1, int
...
 
+public void Confirm_FunctionLuck(int client)
+{
+	char buffer[256];
+	Menu menu = new Menu(Menu_ConfirmTryLuck);
+
+	FormatEx(buffer, sizeof(buffer), "%t\n", "confirm_luck", g_hLuckCredits.IntValue);
+	menu.SetTitle(buffer);
+
+	FormatEx(buffer, sizeof(buffer), "%t", "Yes");
```
Comment (R1KO) [inline addons/sourcemod/scripting/shop/functions.sp:295]: `FormatEx(buffer, sizeof(buffer), "%T", "Yes", client);`

File: addons/sourcemod/scripting/shop/functions.sp [hunk 3]
```diff
@@ -284,6 +284,47 @@ public int Functions_Menu_Handler(Menu menu, MenuAction action, int param1, int
 	}
 }
 
+public void Confirm_FunctionLuck(int client)
+{
+	char buffer[256];
+	Menu menu = new Menu(Menu_ConfirmTryLuck);
+
+	FormatEx(buffer, sizeof(buffer), "%t\n", "confirm_luck", g_hLuckCredits.IntValue);
```
Comment (R1KO) [inline addons/sourcemod/scripting/shop/functions.sp:292]: add space after `\n`

File: addons/sourcemod/scripting/shop/functions.sp [hunk 4]
```diff
@@ -284,6 +284,47 @@ public int Functions_Menu_Handler(Menu menu, MenuAction action, int param1, int
...
+	menu.SetTitle(buffer);
+
+	FormatEx(buffer, sizeof(buffer), "%t", "Yes");
+	menu.AddItem("a", buffer);
+
+	FormatEx(buffer, sizeof(buffer), "%t", "No");
+	menu.AddItem("b", buffer);
```
Comment (R1KO) [inline addons/sourcemod/scripting/shop/functions.sp:299]: why `a` and `b` ?

**Review by R1KO — CHANGES_REQUESTED** [review]

**Comment by oylsister** [comment]
I have closed this pull request due to reworking confirm menu has been done

**Pull request closed without merging**